### DISCUSSION TITLE
Recommended editor/compiler for argoCD

### DISCUSSION THREAD

**Discussion by BenSyntio** [argoproj/argo-cd#6184] [Q&A] [answered]
While learning argoCD i get compilation errors and it is hard to solve them since I don't know about any good editor for argoCD.
I'm using VSCode at the moment. Can someone recommend either an editor or a plugin for VSCode that can help with compilation errors in argoCD

**Comment by jgwest** [comment, marked as answer]
Hi @BenSyntio, Argo CD development shouldn't require any special tools beyond those used for traditional Go development. Personally I use VS Code with the Go plugin from 'Go Team at Google', available from the VS Code Marketplace.

**Reply by BenSyntio** [reply]
Thank you for a quick answer!
